### ISSUE TITLE
missing StandardScaler affecting GBT Cross-Val Score

### ISSUE THREAD

**Issue by allaccountstaken** [learn-co-curriculum/dsc-gradient-boosting-lab#1]
Dear Team, 
we are missing StandardScaler import from sklearn. Scaling of the data (X) is needed to match expected output from cross validation for GBT algorithm.

Cheers,

**Comment by isobeldaley** [comment]
This issue has still not been resolved

**Comment by sumedh10** [comment]
It's not necessary to scale data when using tree-based models. The variation in output was due to missing `random_state`, which has now been fixed.
